join_request/ermalkaleci.yml

```
@@ -0,0 +1,22 @@
+collective: Potoc
+member:
+  name: Ermal Kaleci
+  address: 14iKbZws1fjJ6TH27yoRq6KeeVNof83VmxUBN2W2udQVBe5o
+  github: ermalkaleci
```

please fill your matrix handle as per https://github.com/super-collective/collective-cli/pull/4#discussion_r1586380064
```suggestion
  address: 14iKbZws1fjJ6TH27yoRq6KeeVNof83VmxUBN2W2udQVBe5o
  matrix:
  github: ermalkaleci
```